xwiki-commons-core/xwiki-commons-diff/xwiki-commons-diff-display/src/main/java/org/xwiki/diff/display/internal/DefaultUnifiedDiffDisplayer.java

```diff
@@ -341,13 +352,14 @@ public void setLastConflict(Conflict<E> lastConflict)
     private <E> boolean canDeltaSplitted(Conflict<E> conflict, Delta<E> delta)
     {
         if (conflict != null) {
-            // a delta can be splitted only if one of its chunk is > 1
+            // a delta can be splitted only if it concerns a change
```

Comment:
Is the old comment `a delta can be splitted only if one of its chunk is > 1` deprecated? Reading the code it looks like it is still relevant (in addition to your new comment).

PS: split is an irregular word.

Reply:
Indeed, thanks. Also fixed the typo.